### PR TITLE
[CALCITE-2093, CALCITE-2088] More tests, removing locale dependent data from tests

### PR DESCRIPTION
**Pull request by ptrbojko**
Additional changes:
- Hinting gititgnore to ignore netbeans project config overrides

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -411,6 +411,11 @@ limitations under the License.
         <artifactId>hsqldb</artifactId>
         <version>${hsqldb.version}</version>
       </dependency>
+		<dependency>
+			<groupId>org.hsqldb</groupId>
```
Comment (vlsi) [inline pom.xml]: Would you please use spaces instead of tabs for indentation like used in the file?

File: plus/pom.xml [hunk 2]
```diff
@@ -81,8 +81,23 @@ limitations under the License.
...
+    <dependency>
+      <groupId>org.hsqldb</groupId>
+      <artifactId>sqltool</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>net.hydromatic</groupId>
+      <artifactId>quidem</artifactId>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.commons</groupId>
```
Comment (vlsi) [inline plus/pom.xml]: Please remove 2 extra spaces

File: pom.xml (hunk 1 above)
Comment (ptrbojko) [reply pom.xml]: done

File: plus/pom.xml (hunk 2 above)
Comment (ptrbojko) [reply plus/pom.xml]: done

**Comment by ptrbojko** [comment]
Travis fails because https://github.com/julianhyde/chinook-data-hsqldb is not in Maven Central.

**Pull request closed without merging**